src/site/xdoc/building.xml

```diff
@@ -24,8 +24,8 @@ limitations under the License.
 <!-- ================================================== -->
 <section name="Overview">
 <p>
-  Commons Lang uses <a href="http://maven.apache.org">Maven</a> or
-  <a href="http://ant.apache.org">Ant</a> as a build system.
+  Commons Lang uses <a href="https://maven.apache.org">Maven</a> or
+  <a href="https://ant.apache.org">Ant</a> as a build system.
```

Comment:
-1: We do not use Ant, only Maven.

Reply:
This PR is primarily about fixing the links. But I removed the reference to Ant.